prow/cluster/jobs/istio/bots/istio.bots.yaml

```diff
@@ -0,0 +1,54 @@
+presubmits:
+
+  istio/bots:
+
+    - name: bots-lint
+      path_alias: istio.io/bots
+      decorate: true
+      always_run: true
```

Comment:
FYI, when configuring new jobs we often initially specify `skip_report: true` to prevent the jobs from actually reporting to GH. They still run automatically on PRs, but do so silently. This is a good way to get signal that the jobs are working as intended and once that is validated we remove the `skip_report: true`.

This isn't necessary, especially if there is high confidence that the jobs will WAI or a small number of developers impacted, but it is a handy pattern for trying out new jobs without risking negative impact on developers so I thought I'd mention it.

Reply:
Oh,  that's indeed handy. I think in the case we're fine since there are few developers affected and I'm pretty sure this will just work. But definitely for future changes.